mantidproject/mantid
# ISIS Powder system test spline tolerance

This should fix the HRPD system test failures seen on the build servers as well as enable the ISIS_PowderPearl system tests. The work has involved replacing some out of date reference files as well as increasing/adding tolerance in tests that involve creating splines of the data. 

**To test**
* Ensure all ISIS_Powder system tests pass locally (Ideally this should be checked for multiple operating systems) - These passed locally for me on windows 10 and Ubuntu 16.04
* Ensure system tests pass on build server (RHEL 7)


Fixes #21275 #21287

**Release Notes** 
*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
